src/utils/address.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  checkAddressChecksum,
+  toChecksumAddress,
+  isAddress,
+  isHexStrict,
+} from 'web3-utils';
+
+export function getAddressWithoutNetworkPrefix(address = ''): string {
```

Comment:
Mixed use of arrow and classic functions in this file

Reply:
Done!